### PR TITLE
big-refactor pull request

### PR DESCRIPTION
**Pull request by unholy-codeism**
~~DO NOT MERGE !
I have opened this for the sake of having it open - and also to allow for cleaner, better code reviews from other contributors.~~
=> I marked it as a draft.

**Tasklist** : 

- [x] Refactor commands
- [x] Manually fix each minigame to adapt to the changes 
- [ ] Refactor EconomySystem
- [ ] Most likely, implement the typical command handler
- [ ] Async functions? (see #62, ***I need explicit confirmation that async functions won't be done on another PR***)
- [ ] Other things that will come as I find more things that can be fixed

The first 7 commits only move stuff around (constants, imports, and files). This is only temporary as I will start implementing your average discord.js command handler over time (even though this one works just fine).

~~I'll add what the other commits do as I go along and update this PR.~~ -> not needed you guys are smart enough to read the commit messages (i try to keep them at least somewhat consistent)

### CODE REVIEW

**Comment by Felix-44** [comment]
since this is a huge thing, I am sorry to say I won't be able to help much (have my stuff to finish first)
either way, good luck!

> ~DO NOT MERGE !~
_it's a draft we can't merge it_

**Comment by unholy-codeism** [comment]
alright, thanks for chiming in 
i'd like to know @LJMSU and @Volas171 (volas is doing #60 tho i wonder if this pr won't break his changes) 's opinions

**Comment by LJMSU** [comment]
- I don't think `math.js` is necessary. Maybe move `&dice` and `&clown` into `jokes.js`, and `&eval` should move somewhere else, preferably `evaluators.js` where `&debug` and `&lua` (?) can also be put there.
- `Debugging` should be `isDebugging` because it sounds more descriptive (in my opinion).

**Comment by Felix-44** [comment]
are we sure we need this refactor btw

**Comment by unholy-codeism** [comment]
> are we sure we need this refactor btw

probs better to do it now 
+ it probably will offer some performance gain once i reach the economysystem in terms of changes
these first commits don't looks very necessary but they'll be essential later on

**Comment by Felix-44** [comment]
> > are we sure we need this refactor btw
> 
> probs better to do it now + it probably will offer some performance gain once i reach the economysystem in terms of changes these first commits don't look very necessary but they'll be essential later on

I mean yes, but there are many branches that will soon be merged, you'll have to update your branch every time

**Comment by unholy-codeism** [comment]
> > > are we sure we need this refactor btw
> > 
> > 
> > probs better to do it now + it probably will offer some performance gain once i reach the economysystem in terms of changes these first commits don't look very necessary but they'll be essential later on
> 
> I mean yes, but there are many branches that will soon be merged, you'll have to update your branch every time



and? i have seen worse

**Comment by unholy-codeism** [comment]
\+ i can do everything from cli lmao
it's very fast

**Comment by Felix-44** [comment]
just..good luck
you will really need it

**Comment by unholy-codeism** [comment]
> * I don't think `math.js` is necessary. Maybe move `&dice` and `&clown` into `jokes.js`, and `&eval` should move somewhere else, preferably `evaluators.js` where `&debug` and `&lua` (?) can also be put there.
> 
>     * `Debugging` should be `isDebugging` because it sounds more descriptive (in my opinion).

alright, i'll be doing a commit with these changes in the next 10 minutes

**Comment by Felix-44** [comment]
imo you're making the files too messy, I'd keep every math command inside math.js

**Comment by unholy-codeism** [comment]
not wrong but not correct either
i think LJ is right, i'll do that

**Comment by unholy-codeism** [comment]
> not wrong but not correct either i think LJ is right, i'll do that

ended up not doing that; won't

File: src/Minigames/crew.js [hunk 1]
```diff
@@ -131,13 +136,14 @@ Commands.crew = new Command("Find the imposter!\n\n" + Amongus.help + "\npossibl
     }
     Amogus.turns = Math.max(Amogus.turns - 1, 0)
     if (Amogus.turns == 0) {
-        message.channel.send("The impostor killed you!\nThe impostor was " + Amogus.getSussier() + ".\nGame over.")
+        message.channel.send("The impostor killed you!\nThe impostor was " + Amogus.getSussiest() + ".\nGame over.")
         let EconomySystem = Economy.getEconomySystem(message.author)
         EconomySystem.steal(30 + (10 * aturn), message)
-        Amogus.reset()
+        delete Amogus
```
Comment (LJMSU) [inline src/Minigames/crew.js:142]: `delete` does not delete a variable.

**Review by LJMSU — CHANGES_REQUESTED** [review]

File: src/Minigames/crew.js (hunk 1 above)
Comment (Felix-44) [reply src/Minigames/crew.js:142]: wait wtf, why delete even if?
it should reset the values to default, not straight up delete it...?

**Comment by Volas171** [comment]
> alright, thanks for chiming in i'd like to know @LJMSU and @Volas171 (volas is doing #60 tho i wonder if this pr won't break his changes) 's opinions

it wont as it already got broke

File: src/Minigames/crew.js (hunk 1 above)
Comment (unholy-codeism) [reply src/Minigames/crew.js:142]: fair 
I wasn't sure if should be doing that, lemme rollback

**Comment by Volas171** [comment]
wait what was that

**Pull request closed without merging**